user_docs/en/changes.t2t

```diff
@@ -41,8 +40,8 @@ What's New in NVDA
 - Pressing sp2 on an ALVA BC6 display to emulate the alt key now works as advertised. (#8360)
 - NVDA no longer announces redundant keyboard layout changes. (#7383)
 - Mouse tracking is now much more accurate in Notepad and other plain text edit controls when in a document with more than 65535 characters. (#8397)
-- NVDA will recognize more dialogs in Windows 10 and other modern applications. (#7473)
-- On windows 10 rs5 and above, NVDA no longer fails to track the system focus when an application freezes or floods the system with events. (#8539)
+- NVDA will recognize more dialogs in Windows 10 and other modern applications. (#8405)
+- On Windows 10 rs5 and Server 2019 and above, NVDA no longer fails to track the system focus when an application freezes or floods the system with events. (#7345, #8535)

```

Comment:
I wonder whether we should refer to rs5 as the Windows 10 Oktober 2018 update, which will most likely be the name for that update.